Code/Common/include/sitkImage.h

```diff
@@ -254,6 +255,32 @@ namespace simple
     /** Transform continuous index to physical point */
     std::vector< double > TransformContinuousIndexToPhysicalPoint( const std::vector< double > &index) const;
 
+    /** \brief Interpolate pixel value at a continuous index.
+     *
+     * This method is not supported for Label pixel types.
+     *
+     * The valid range of continuous index is [-0.5, size-0.5] for each dimension. An exception is thrown if index is out of bounds.
+     *
+     * @param index The continuous index must be at least the length of the image dimension.
+     * @param interp The interpolation type to use, only sitkNearest and sitkLinear are supported for Vector and Complex pixel types.
+     *
+     * @return All supported pixel types are returned as an array, where complex number are returned with the real followed by the complex component.
+     */
+    std::vector<double> EvaluateAtContinuousIndex( const std::vector<double> &index, InterpolatorEnum interp = sitkLinear) const;
+
+    /** Interpolate pixel value at a physical point.
+     *
+     * This method is not supported for Label pixel types.
+     *
+     * An exception is thrown if the point is out of the defined region for the image.
+     *
+     * @param point The physical point which is transformed to a continuous index for interpolation.
```

Comment:
Possibly change to "The physical point at which the interpolation is computed."
That the physical point is converted to a continuous index is an implementation detail.